docs/examples/opentelemetry-example-app/src/opentelemetry_example_app/flask_example.py

```diff
@@ -34,15 +34,15 @@
 trace.set_tracer_provider(TracerProvider())
 
 opentelemetry.ext.requests.RequestsInstrumentor().instrument()
-FlaskInstrumentor().instrument()
 
 trace.get_tracer_provider().add_span_processor(
     SimpleExportSpanProcessor(ConsoleSpanExporter())
 )
 
-
 app = flask.Flask(__name__)
 
```

Comment:
Not related to this PR, but is this example missing `app.run()`, how am I supposed to run it?